lab/lab1/part3-data-transformation.html

```diff
@@ -70,9 +70,28 @@
     Start code to filter data
 
     ===================== */
-
-
-
+//for clean data
```

Comment:
Nothing appears on the map when I load the page:
part3-data-transformation.html



![image](https://user-images.githubusercontent.com/8103418/52543934-9a08b500-2d7b-11e9-960d-0159bac30172.png)


Does it work for you?

Reply:
Changed. It should work now.